sr_example/scripts/sr_example/hand_examples/sr_right_hand_follow_finger_trajectory.py

```diff
@@ -59,26 +55,30 @@
 waypoints.append(start_pose)
 
 # Move vertically
-pose = geometry_msgs.msg.Pose()
-pose.position.x = 0.033
-pose.position.y = -0.016
-pose.position.z = 0.190
-pose.orientation.w = 1.00
-waypoints.append(pose)
-
-pose = geometry_msgs.msg.Pose()
-pose.position.x = 0.033
-pose.position.y = -0.035
-pose.position.z = 0.178
-pose.orientation.w = 1.00
-waypoints.append(pose)
-
-pose = geometry_msgs.msg.Pose()
-pose.position.x = 0.033
-pose.position.y = -0.068
-pose.position.z = 0.151
-pose.orientation.w = 1.00
-waypoints.append(pose)
+pose1 = geometry_msgs.msg.Pose()
+pose1.position.x = 0.033
+pose1.position.y = -0.016
+pose1.position.z = 0.190
+pose1.orientation.w = 1.00
+waypoints.append(pose1)
+
+pose2 = geometry_msgs.msg.Pose()
+pose2.position.x = 0.033
+pose2.position.y = -0.035
+pose2.position.z = 0.178
+pose2.orientation.w = 1.00
+waypoints.append(pose2)
+
+pose3 = geometry_msgs.msg.Pose()
+pose3.position.x = 0.033
+pose3.position.y = -0.068
+pose3.position.z = 0.151
+pose3.orientation.w = 1.00
+waypoints.append(pose3)
+
+waypoints.append(pose2)
```

Comment:
I created a trajectory first moving horizontally (abduction/adduction) of the finger and then moving vertically (flexing the finger) so to do the latter I appended three poses and then, to move back to the initial pose, I appended them but in reverse order.